tests/test_skyWcs.py

```diff
@@ -694,6 +695,10 @@ def testBasics(self):
         makeSkyWcs(self.metadata, strip=True)
         self.assertEqual(len(self.metadata.names(False)), 0)
 
+    def testBasicsStrip(self):
+        stripWcsMetadata(self.metadata)
+        self.assertEqual(len(self.metadata.names(False)), 0)
```

Comment:
We should add a test here to strip again and make sure it returns the same thing.